lib/Dialect/ESI/runtime/setup.py

```diff
@@ -0,0 +1,124 @@
+# Build/install the pycde python package. Blatantly copied from npcomp.
```

Comment:
nit: this comment should maybe be updated to mention "ESI Runtime python package" or something. npcomp is a real throwback!